### DISCUSSION TITLE
idea: sfm as a posible "game"

### DISCUSSION THREAD

**Discussion by Sans353CZ** [SentientTurtle/RBXLX2VMF#7] [Ideas]
im trying to get roblox maps into source filmmaker but i cant find anything (idk how to do it my self)

**Comment by SentientTurtle** [comment]
Source Filmmaker requires 'compiled' .bsp maps, this tool generates 'uncompiled' .vmf maps.

You will need to compile the map in hammer (or hammer++) for it to be useable in SFM. The 'game' selection applies to the game configuration used in hammer, and is used to set a default skybox.

For SFM, the 'tf2' configuration should provide reasonable defaults, if not you'll need to change the skybox in hammer, to one supported by SFM. There are steam community guides that go into detail about SFM's quirks.

**Reply by Sans353CZ** [reply]
ok
